### DISCUSSION TITLE
Control Change Numbers

### DISCUSSION THREAD

**Discussion by fenderf4i** [tttapa/Control-Surface#1105] [Q&A]
Hi! I'm trying to make a Midi footswitch to work with a guitar amp modeller (Line 6 Helix), and am trying to figure out how to send control change numbers like CC49 to it. The following is what I'm trying to use but can't figure out how to just send a straight CC49 code rather than something like General_Purpose_Controller_5.

`#include <Control_Surface.h>

// Instantiate a MIDI over USB interface
USBMIDI_Interface midi;
using namespace MIDI_Notes;

// Button Rows

CCButton button1 { 2, {MIDI_CC::General_Purpose_Controller_5, Channel_4},};
CCButton button2 { 3, {MIDI_CC::General_Purpose_Controller_6, Channel_4},};
CCButton button3 { 4, {MIDI_CC::General_Purpose_Controller_7, Channel_4},};
CCButton button4 { 5, {MIDI_CC::General_Purpose_Controller_8, Channel_4},};
CCButton button5 { 6, {MIDI_CC::Balance, Channel_4},};
CCButton button6 { 7, {MIDI_CC::Pan, Channel_4},};

NoteButton button7  = { 8, {note( A, 2), CHANNEL_4}};

void setup() {
    // Initialize everything
    Control_Surface.begin();
}

void loop() {
    // Update the Control Surface (check whether the potentiometer's input has 
    // changed since last time, if so, send the new value over BLE MIDI).
    Control_Surface.loop();
}`

**Comment by tttapa** [comment]
You can just use the number directly. Please see https://tttapa.github.io/Control-Surface/Doxygen/d3/df7/midi-tutorial.html#midi_md-midi-addresses.
